### DISCUSSION TITLE
SSL Errors?

### DISCUSSION THREAD

**Discussion by TheeKDubya** [cwendt94/espn-api#520] [ESPN API] [answered]
Today I tried running my usual weekly scripts and I was unable to connect, is anyone else getting this?

Exception has occurred: SSLError
HTTPSConnectionPool(host='fantasy.espn.com', port=443): Max retries exceeded with url: /apis/v3/games/ffl/seasons/2023/segments/0/leagues/[my league id]?view=mTeam&view=mRoster&view=mMatchup&view=mSettings&view=mStandings (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self-signed certificate in certificate chain (_ssl.c:1006)')))
ssl.SSLCertVerificationError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self-signed certificate in certificate chain (_ssl.c:1006)

During handling of the above exception, another exception occurred:

urllib3.exceptions.SSLError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self-signed certificate in certificate chain (_ssl.c:1006)

The above exception was the direct cause of the following exception:

urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='fantasy.espn.com', port=443): Max retries exceeded with url: /apis/v3/games/ffl/seasons/2023/segments/0/leagues/[my league id]?view=mTeam&view=mRoster&view=mMatchup&view=mSettings&view=mStandings (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self-signed certificate in certificate chain (_ssl.c:1006)')))

**Comment by cwendt94** [comment]
This is most likely something wrong with their SSL Cert. When you try again, does it still happen?

**Reply by TheeKDubya** [reply, marked as answer]
Interestingly it works fine if I use a different laptop, so it must be something local. Still doesn't work on the other laptop. Thanks for the response, I'll look into it.
